src/main/java/org/folio/service/transactions/DefaultTransactionService.java

```diff
@@ -3,24 +3,33 @@
 import io.vertx.core.Future;
 import io.vertx.core.Promise;
 import io.vertx.ext.web.handler.HttpException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.folio.rest.core.model.RequestContext;
 import org.folio.rest.jaxrs.model.Transaction;
 import org.folio.rest.jaxrs.resource.FinanceStorageTransactions;
 import org.folio.rest.persist.PgUtil;
 
 public class DefaultTransactionService extends AbstractTransactionService {
 
+  private static final Logger logger = LogManager.getLogger(DefaultTransactionService.class);
+
   @Override
   public Future<Transaction> createTransaction(Transaction transaction, RequestContext requestContext) {
+    logger.debug("createTransaction:: Trying to create transaction");
     Promise<Transaction> promise = Promise.promise();
     PgUtil.post(TRANSACTION_TABLE, transaction, requestContext.getHeaders(), requestContext.getContext(), FinanceStorageTransactions.PostFinanceStorageTransactionsResponse.class, event -> {
       if (event.succeeded()) {
         if (event.result().getStatus() == 201) {
-          promise.complete((Transaction) event.result().getEntity());
+          Transaction createdTransaction = (Transaction) event.result().getEntity();
+          logger.info("createTransaction:: Transaction with id {} successfully created", createdTransaction.getId());
+          promise.complete(createdTransaction);
         } else {
+          logger.error("createTransaction:: Creating transaction failed with status {}", event.result().getStatus());
```

Comment:
in at least error logs we need to add transaction id.
In this case we can search what is going wrong with just knowing this transactionId

Reply:
added